pkg/templates/images/images.go

```diff
@@ -377,26 +373,26 @@ func optionalResources() map[Resource]map[string]string {
 		WeaveNetCNINPC:  {"*": "docker.io/weaveworks/weave-npc:2.8.1"},
 
 		// Calico VXLAN
-		CalicoVXLANCNI:        {"*": "quay.io/calico/cni:v3.28.2"},
-		CalicoVXLANController: {"*": "quay.io/calico/kube-controllers:v3.28.2"},
-		CalicoVXLANNode:       {"*": "quay.io/calico/node:v3.28.2"},
+		CalicoVXLANCNI:        {"*": "quay.io/calico/cni:v3.29.2"},
+		CalicoVXLANController: {"*": "quay.io/calico/kube-controllers:v3.29.2"},
+		CalicoVXLANNode:       {"*": "quay.io/calico/node:v3.29.2"},
 
 		// Cilium
-		Cilium:         {"*": "quay.io/cilium/cilium:v1.16.3@sha256:62d2a09bbef840a46099ac4c69421c90f84f28d018d479749049011329aa7f28"},
-		CiliumOperator: {"*": "quay.io/cilium/operator-generic:v1.16.3@sha256:6e2925ef47a1c76e183c48f95d4ce0d34a1e5e848252f910476c3e11ce1ec94b"},
-		CiliumEnvoy:    {"*": "quay.io/cilium/cilium-envoy:v1.29.9-1728346947-0d05e48bfbb8c4737ec40d5781d970a550ed2bbd@sha256:42614a44e508f70d03a04470df5f61e3cffd22462471a0be0544cf116f2c50ba"},
+		Cilium:         {"*": "quay.io/cilium/cilium:v1.17.1@sha256:8969bfd9c87cbea91e40665f8ebe327268c99d844ca26d7d12165de07f702866"},
```

Comment:
The chart version in Kustomization is still at v1.16.3. Is that intentional?

Reply:
Updated